### ISSUE TITLE
why `pred.exp()`?

### ISSUE THREAD

**Issue by sosuneko** [chenjun2hao/DDRNet.pytorch#14]
Thanks for the nice implementation :cat: :100:, I have a small question:

https://github.com/chenjun2hao/DDRNet.pytorch/blob/bc0e193e87ead839dbc715c48e6bfb059cf21b27/lib/datasets/base_dataset.py#L222 
Why `pred.exp()` in inference? The real scores should be `softmax(pred, dim=1)` right? Just wonder what is the usage of adding `exp()` here, is it for visualization?
